packages/contracts/contracts/MarketRegistry.sol

```diff
@@ -81,7 +83,7 @@ contract MarketRegistry is
 
     event MarketCreated(address indexed owner, uint256 marketId);
     event SetMarketURI(uint256 marketId, string uri);
-    event SetPaymentCycleDuration(uint256 marketId, uint32 duration);
+    event SetPaymentCycleValue(uint256 marketId, uint32 duration);
```

Comment:
lets keep the old event as well as the new one so that the generated ABI still includes it, so that subgraph will be able to pick up the old events that were emitted + the new ones